scipy/signal/tests/test_spectral.py

```diff
@@ -122,7 +122,7 @@ def test_window_external(self):
         fe, pe = periodogram(x, 10, win)
         assert_array_almost_equal_nulp(p, pe)
         assert_array_almost_equal_nulp(f, fe)
-
+        
```

Comment:
Looks like unnecessary whitespace. Weird that the pep8/pyflakes check didn't catch this.